### PR TITLE
[FEAT] Adding fruit quest UI

### PR DESCRIPTION
**Pull request by HBandesh**
# Description

In this PR we have added the UI for displaying the fruit quest. 
image
image

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/island/farmerQuest/FarmerQuest.tsx [hunk 1]
```diff
@@ -4,43 +4,70 @@ import { GRID_WIDTH_PX, PIXEL_SCALE } from "features/game/lib/constants";
...
-            If you prove you are a hardworking Bumpkin, I will give you rare
-            Farmer clothing.
-          </p>
-          <Button disabled onClick={() => setState("progress")}>
-            Continue
-          </Button>
+  const ModalDescription = () => {
+    return (
+      <>
+        <p className="mb-3">
```
Comment (hujikol) [inline src/features/island/farmerQuest/FarmerQuest.tsx:20]: Can we replace the jsx fragment into div to wrap the long text and add more margin?

**Review by adamhannigan — COMMENTED** [review]
How should we hide this quest until the wearables are ready?

**Comment by HBandesh** [comment]
@adamhannigan , I have added a feature flag for the quest. So this quest should only be available on the testnet now.

**Review by c-py — DISMISSED** [review]
Tested and working well!

**Review by c-py — APPROVED** [review]
LGTM!

**Pull request merged**